core/src/main/java/com/github/mizool/core/rest/errorhandling/ErrorHandler.java

```diff
@@ -75,6 +77,10 @@ else if (isAssignable(ClientErrorException.class, cursor))
             {
                 result = handleClientError((ClientErrorException) cursor);
             }
+            else if (isAssignable(RuleViolationException.class, cursor))
```

Comment:
move this up so it's directly below the `ConstraintViolationException` stuff to which it is conceptually similar. incidentally, the order of `if` branches then conforms to the order of the methods they call.